agent/reaper/reaper_stub.go

```diff
@@ -7,6 +7,6 @@ func IsInitProcess() bool {
 	return false
 }
 
-func ForkReap(opt ...Option) error {
```

Comment:
This trips up the linter on my box, so I fixed.  Missed by CI because CI linting happens on linux.